services/api/src/resources/env-variables/resolvers.ts

```diff
@@ -21,18 +13,35 @@ const envVarScopeToString = R.cond([
   [R.T, R.identity],
 ]);
 
-const getEnvVarsByProjectId = async (
+export const getEnvVarsByProjectId = async (
   { id: pid },
   args,
   {
     sqlClient,
     hasPermission,
+    models,
   },
 ) => {
   await hasPermission('env_var', 'project:view', {
     project: pid,
   });
-
+  let queries = [];
```

Comment:
Since you added the env variables as part of the group itself, and lagoon is internally using that method, can we get rid of this change to `getEnvVarsByProjectId` all-together and only get project env vars from the project field? It seems odd, that this would work, but the `environment` doesn't have a similar `withGroupEnvVars`/`withProjectEnvVars` argument.